app/assets/javascripts/test/enzyme/e2e-setup.js

```diff
@@ -50,6 +52,15 @@ function setCurrToken(token: string) {
   currToken = token;
 }
 
+// The values of these keys change if objects are newly created by the backend
+// They have to be omitted from some snapshots
+const volatileKeys = ["id", "formattedHash", "modified", "created"];
```

Comment:
I'm not sure how easy it is, but, on second thought, I think it would be better to replace the values for these keys with a dummy value instead of omitting them completely. For example, `{id: "id", formattedHash: "formattedHash", ...}`. Then, we would also catch the case in which the fields are completely missing in the response. Not sure, though, whether it's worth the change.